src/players/DesktopArticleVideoPlayer/DesktopArticleVideoPlayer.tsx

```diff
@@ -48,6 +49,7 @@ interface DesktopArticleVideoPlayerProps {
 }
 
 const DesktopArticleVideoPlayer: React.FC<DesktopArticleVideoPlayerProps> = ({ playlist }) => {
+	usePlayerVersion();
```

Comment:
We don't need to put this here or the other one. Add it to the SharedJW component. It'll only be once then